packages/ui-concerto/src/lib/components/fields.js

```diff
@@ -75,18 +75,21 @@ export const ConcertoDateTime = ({
   onFieldValueChange,
   skipLabel,
   type,
-}) => (
+}) => {
+  const date = Date.parse(value) ? value : new Date().toISOString();
+  return (
   <Form.Field required={required}>
     <ConcertoLabel skip={skipLabel} name={field.getName()} />
     <Input
       type={type}
       readOnly={readOnly}
-      value={new Date(value).toISOString().slice(0, 19)}
+      value={date.slice(0, 16)}
```

Comment:
I don't fully understand this PR... and most of it looks fine, but this `slice`call looks quite arbitrary. Can you explain what the purpose / reasoning behind it is?

Reply:
This is just for the form itself, which only allows the user to choose a date. The value supplied by the form on change is already an ISO string.